src/content/platform-integration/web/building.md

```diff
@@ -4,174 +4,125 @@ description: Instructions for creating a Flutter app for the web.
 short-title: Web development
 ---
 
-This page covers the following steps for getting started with web support:
-
-* Configure the `flutter` tool for web support.
-* Create a new project with web support.
-* Run a new project with web support.
-* Build an app with web support.
-* Add web support to an existing project.
+This page provides an overview of how to configure, run and build a web
```

Comment:
Don't make me lecture you on the Oxford comma!

```suggestion
This page provides an overview of how to configure, run, and build a web
```

Reply:
It was an honest mistake I swear!